### DISCUSSION TITLE
How to combine DVC and Gerrit?

### DISCUSSION THREAD

**Discussion by DeqiangXIA** [treeverse/dvc#9772] [Help]
Great work! 
I want to know how to combine DVC and Gerrit? For instance，how can i use 'dvc push' to push a data file to Gerrit? Thanks!

**Comment by daavoo** [comment]
Hi @DeqiangXIA , as far as I understand, Gerrit is not meant to store large data artifacts.
You need to pick and set up one of the available options for DVC remotes (https://dvc.org/doc/start/data-management/data-versioning#storing-and-sharing) and I think you could just track the git-tracked DVC metadata files (`.dvc` / `dvc.lock` ) in Gerrit

**Reply by DeqiangXIA** [reply]
Hi @daavoo, Thanks for your answer. I want to use Gerrit to check the large data artifacts before I put them into my DVC remotes. How can I achieve this when I use `dvc push`? Thanks again!
